src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/classifyIuvActivity.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+/**
+ * Interface for defining an activity to process payment classifications.
```

Comment:
let's specify what does it mean `IUV` classification (i'm proposing something, but change it accordingly to the right logic)
```suggestion
 * Interface for defining an activity to process payment classifications based on IUV and IUD.
```